find_store_github_actions.R

```diff
@@ -0,0 +1,19 @@
+find_store_github_actions <- function(repo,
+                                      github_pattern = "\\.github.*workflows.*(yml|yaml)",
+                                      github_folder_name = "github_actions") {
+
+  if (!dir.exists(github_folder_name)) {
+    invisible(dir.create(github_folder_name, showWarnings = FALSE))
+  }
+
+  files = list.files('tmp', recursive = TRUE, all.files = TRUE)
+
+  if (length(files_found <- grep(github_pattern, files, value = TRUE))) {
```

Comment:
I would simply keep this as a branch in the `seq_along(known_meta)` loop (as well as adding a `'github-actions' = 'github/workflows'` to `known_meta`).

I may re-evaluate later whether it's worth to functionalize a bit more, but I don't see the list of files to scrape growing too much more for now.

The above will already handle the first `dir.create` and skip need to `list.files` twice.

Reply:
I have a couple of workflows defined for example in this [repo](https://github.com/JanCaha/SpatialKDE/tree/master/.github/workflows). One to perform R CMD Check, one to compile and deploy the website and one to build binary release for Windows.

I wanted to change the main script as little as possible, that was the reason to put it in a separate function.  Also, the process is a little bit different due to variable number of workflows.

Reply:
Thanks, understood, I think what you're going for is correct, basically to have structure

```
github-actions/
  pkg1/
    action1.yml
    action2.yml
    ...
  pkg2/
    action1.yml
  ...
```